### PR TITLE
Removed line in composer.json mapping vendor install to app/code

### PR DESCRIPTION
**Pull request by scott-taylor-extend**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,13 +21,5 @@
         "psr-4": {
             "Extend\\Integration\\": ""
```
Comment (jay-faust-extend) [inline composer.json:22]: What about this above line? I know nothing about composer but I would assume there shouldn't be any references to `Extend/Integration`

Comment (scott-taylor-extend) [reply composer.json:22]: That says that it an alias helloextend/integration.  All the module's class names still start with Extend\Integration, according to PSR4 standards

**Review by aarondeane-extend — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (scott-taylor-extend) [reply composer.json:22]: See a similar example here https://github.com/magento/magento2/blob/a047c7639c63bf2b858e0bbd9689eb993a48a5a2/app/code/Magento/Catalog/composer.json#L51

Comment (scott-taylor-extend) [reply composer.json:22]: ^even though in composer, it's installed in the folder magento/module-catalog

**Review by jay-faust-extend — APPROVED** [review]

**Pull request merged**